### DISCUSSION TITLE
Is MQTT Auto Discovery Supported?

### DISCUSSION THREAD

**Discussion by tommycw1** [TD22057/insteon-mqtt#287] [Q&A] [answered]
Hello,

I am transitioning from using the openHAB native 2.x binding to using insteon<->MQTT bridge and have a system with about 50 Insteon devices.  I have it all configured now and can command lights on/off etc via the command line.  I have configured one to work with OH as a demo, however it is a lot of work and was wondering if the insteon<->MQTT Bridge supports auto discovery via the home assistant or homie protocols?  I'm really really hoping to not have to create all of these MQTT "things" and channels in OH.

**Comment by krkeegan** [comment, marked as answer]
Whoa, this is a whole new section of GitHub to me.

See this issue #128.  Support for the discovery protocol is not yet available.  There is a fork in that issue that has support, but it would need to be modified before it could be pulled into the main repository.

**Reply by tommycw1** [reply]
Haha - me too.  I guess I thought it wasn't really an issue to report, really just a question.  Appreciate the response.  Do you know if anyone has used this with openHAB before?

**Comment by krkeegan** [comment]
Yes as of version 0.9.0 MQTT Discovery is now supported.
